requirements.txt

```diff
@@ -1,6 +1,7 @@
 pandas==1.4.3
 psycopg2-binary==2.9.3
 requests==2.28.1
+types-requests==2.28.0
```

Comment:
Do you have to install it locally also?
Why is this preferable vs the old solution of importing typing from the standard library?